src/app/model/User.java

```diff
@@ -36,8 +37,15 @@ public String toString() {
         return this.name + " " + this.lastname;//n
     }
 
-    public boolean equals(String name, String lastname) {
-        return (this.name.equals(name) && this.lastname.equals(lastname));
+    public boolean equals(Object obj) {
+        if (obj instanceof User){
+            User user = (User) obj;
+            return user.getName().equals(this.name) && user.getLastname().equals(this.lastname);
+        }
+        return false;
     }
 
+    public boolean equals(String name, String lastname) {
+        return name.equals(this.name) && lastname.equals(this.lastname);
+    }
 }
```

Comment:
Can you add an extra line at the end? This symbol in Github means the file needs one empty line at the end